Common/Source/Dialogs/dlgRadioSettings.cpp

```diff
@@ -258,11 +270,11 @@ TCHAR Name[250];
     }
 
 
-    if((ValidWayPoint(PassiveRadioIndex)) && 0)
+    if((ValidWayPoint(PassiveRadioIndex)) )
     {
         if( PassiveRadioIndex > RESWP_END )
         {
-          _stprintf(Name,_T("%s"),WayPointList[PassiveRadioIndex].Name);
+        	CopyTruncateString(Name, DEVICE_NAME_LEN, WayPointList[PassiveRadioIndex].Name);
```

Comment:
why do a copy before setCaption ?

Reply:
local copy Name is shorter to add brackets lest and right 
[Stationname] = remote update
>Stationname< = Radio receiving
<Stationname> = Redio sending

Reply:
this : 
          _stprintf(Name,_T("%s"),WayPointList[PassiveRadioIndex].Name);
          if(wpnewPassive)
            wpnewPassive->SetCaption(Name);

is same as
          if(wpnewPassive)
            wpnewPassive->SetCaption(WayPointList[PassiveRadioIndex].Name);
 
so copy is useless ...